### DISCUSSION TITLE
One camera triggerd, multiple cameras recording.

### DISCUSSION THREAD

**Discussion by francoisduprez** [blakeblackshear/frigate#8029] [General]
Is there a possibility in Frigate to have one camera  triggered by a zone and let other camera's start recording.
So if you have a camera covering one aspect of your property you can force all cameras to record while that camera detects motion or other events.

I have one camera in my garden and i don't want to create zones for detection for that camera, because a lot of people walk out of the house in the garden en do garden stuff. So i don't want that camera recording if that happens. 

But, if someone opens the gate and get detected by the gate camera, i want to follow (record) him with the garden for a given time. 

Is this a option, or am i the only one who needs this...

**Comment by NickM-27** [comment]
In 0.13 there is the option for custom events so you could use that to create a custom event. 

There is a pinned request for this camera conditional feature but it is not currently implemented

**Reply by francoisduprez** [reply]
Thanks for the reply,

I installed the beta 2  but i can't find anything about custom events, or the configuration options. Probably i don't understand how to use it..  

I'm trying it in HA, but didn't have the time to look further.  Would be nice if it would be possible with the stand alone app.
